app/src/examples/RuntimeTests/RuntimeTestsExample.tsx

```diff
@@ -2,22 +2,22 @@ import React from 'react';
 import RuntimeTestsRunner from './ReanimatedRuntimeTestsRunner/RuntimeTestsRunner';
 
 // load tests
-import './tests/Animations.test';
+// import './tests/Animations.test';
```

Comment:
Do we want it commented out? 😄

Reply:
Probably yes, this was just an "example suite", which was expected to show the API. This test file will be removed. Probably we are going to provide some test of testing frameworks instead